hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/validate/UnsupportedOperationVisitor.java

```diff
@@ -266,6 +268,7 @@ public Void visit(SqlDataTypeSpec type) {
         }
     }
 
+    @SuppressWarnings("all")
```

Comment:
We should suppress a specific warning, not `all`. But I don't have any IDE or checkstyle warnings if I remove this, maybe it's findbugs one...